src/Razor/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptor.cs

```diff
@@ -29,6 +29,8 @@ protected BoundAttributeDescriptor(string kind)
 
         public bool IsBooleanProperty { get; protected set; }
 
+        public string DefaultValue { get; protected set; }
```

Comment:
Could we make these non-abstract? I know precedence, but I'm not sure it makes a difference.

Reply:
Also, if I'm not mistaken, there's a BoundAttributeDescriptorComparer. It might need to be updated to include this property.